source/sass/patterns/atoms/image.scss

```diff
@@ -7,11 +7,6 @@
 .image {
   @include block-spacing($start: $baselinegrid-space-medium, $end: $baselinegrid-space-medium);
   @include border();
-  @include padding($image-inline-spacing, inline);
-  @include padding($baselinegrid-space-extra_small, block);
+  @include padding($baselinegrid-space-extra_small $image-inline-spacing);
   display: block;
-
-  .figure__asset & {
```

Comment:
Image should have no knowledge of Figure.

Reply:
I think `image` pattern styles should sit in the `image` style file. As discussed, a new CSS class name to replace Image's explicit knowledge of Figure here is the preferred approach.

Reply:
Tried to find a compromise in https://github.com/libero/pattern-library/pull/100/commits/4bf3a9c72d7c0a8f649ed9e73c9d28aff55668f8. It's more complicated than first thought as Figure doesn't know what type of asset is there, so can't add an Image class.